packages/mint-components/src/components/tax-and-cash/sqm-tax-document-submitted/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,77 @@
+
+    Examples: 
+      | documentType |
+      | W9           |
+      | W8-BEN       |
+      | W8-BEN-E     |
+
+  @minutia @ui
+  Scenario Outline: The date submitted and status of a Tax Form is displayed to the user
+    Given the Tax Document is currently <status>
+    When they view the Tax Document Submitted
+    Then they see a badge with <status> text and a respective <badgeVariant>
+    Then they see a message indicating "<taxStatusMessage> on <dateSubmitted>"
+
+    Examples: 
+      | status       | badgeVariant | taxStatusMessage                                                          | dateSubmitted |
+      | ACTIVE       | success      | Submitted on                                                              | Jan 17, 2024  |
+      | NOT_VERIFIED | neutral      | Awaiting Review. Submitted on                                             | Jan 17, 2024  |
+      | NOT_ACTIVE   | danger       | Ensure your information matches your profile and resubmit a new document. | Jan 17, 2024  |
+      | EXPIRED      | danger       | Expired on                                                                | Jan 17, 2024  |
+
+  @minutia @ui
+  Scenario: Status badge and text does not appear if participant is not required to submit tax form
+    When a tax document is not required
+    Then tax document header does not include a document type
+    And the description text under the header displays "Tax documents are only required if you are based in the US. If your country of residence has changed, please contact Support"
+    And the "Submit new document" is not available
+
+  @minutia @ui
+  Scenario Outline: A Danger Alert is displayed if the users tax form is invalid
+    Given the <status> is "NOT_ACTIVE" or "EXPIRED"
+    Then a danger alert indicating the <documentType> with a <taxAlertHeader> and <taxAlertMessage> appears
+
+    Examples: 
+      | status     | documentType | taxAlertHeader                                                             | taxAlertMessage                      |
+      | NOT_ACTIVE | W9           | Your W9 tax form has personal information that doesn't match your profile. | Please resubmit a new W9 form.       |
+      | EXPIRED    | W8-BEN-E     | Your W8-BEN-E tax form has expired.                                        | Please resubmit a new W8-BEN-E form. |
+
+  @minutia @ui
+  Scenario Outline: A Warning Alert is displayed if the users tax form is expiring soon
+    Given the <status> is ACTIVE
+    And the <dateExpired> is 30 days away from the current date
+    Then a warning alert indicating the <documentType> with a <taxAlertHeader>, <taxAlertMessage>, and <dateExpired> appears
+
+    Examples: 
+      | status | documentType | taxAlertHeader              | dateExpired    | taxAlertMessage                |
+      | ACTIVE | W9           | Your W9 tax form expires on | Dec 30th, 2024 | Please resubmit a new W9 form. |
+
+  @unknown
+  Scenario: Submit New Tax Document Form
```

Comment:
I don't think they should go back into the step flow but rather a page with all the docusign things but its specific for coming back later. I believe there are designs in figma now